picard/const/__init__.py

```diff
@@ -142,7 +142,7 @@
 }
 
 # Default query limit
-QUERY_LIMIT = 25
+QUERY_LIMIT = 50
```

Comment:
May be rename it to `DEFAULT_QUERY_LIMIT` and drop comment above (or replace it with an explanation).

Reply:
I think than we can also drop the constant alltogether, I just kept it in case someone uses it for own requests in custom plugins.